qiskit/providers/fake_provider/fake_1q.py

```diff
@@ -32,7 +32,7 @@ def __init__(self):
         configuration = BackendProperties(
             backend_name="fake_1q",
             backend_version="0.0.0",
-            n_qubits=1,
+            num_qubits=1,
```

Comment:
I actually don't think either field is required in `BackendProperties`, this just gets added the payload as an extra field but afaict nothing actually will use this because it's not part of the schema.